src/lib.js

```diff
@@ -1,8 +1,11 @@
+// @ts-expect-error no type declarations
+import bytecode from '../gen/wasm_bg.wasm'
```

Comment:
I think I failed to provide enough details earlier. We generate `../gen/wasm_bg.js` containing contents of the above file in base64 encoded string:

https://github.com/web3-storage/fr32-sha2-256-trunc254-padded-binary-tree-multihash/blob/95cb375bbb2a5d8610191598a306d9ee97320335/scripts/build.js#L15-L25

What I was proposing is to import that module here and then turn that base64 encoded string into buffer to pass into `load`. Which is kind of what we do here 

https://github.com/web3-storage/fr32-sha2-256-trunc254-padded-binary-tree-multihash/blob/95cb375bbb2a5d8610191598a306d9ee97320335/template/wasm.js#L8-L22
But we don't have to use data URIs, in fact as you can see in the commented code we used to use blobs although I think there was some issue with that in AWS

Reply:
Oh ok, I see. That makes sense! I was not understanding what was the file.

Somehow, CF vails to deploy suggested code though:

This `index.js` works:

```js
import bytecode from '../gen/wasm_bg.wasm'

import load, { create } from "../gen/wasm.js"
import { code, CODE_LENGTH, HEIGHT_SIZE, ROOT_SIZE, MAX_SIZE } from './constant.js'
export * from "./type.js"
export { code, CODE_LENGTH, HEIGHT_SIZE, ROOT_SIZE, MAX_SIZE }

await load(bytecode)

export { create }
```

this fails:
```js
import * as base64 from 'multiformats/bases/base64'
import byteEncodedString from '../gen/wasm_bg.js'

import load, { create } from '../gen/wasm.js'
import { code, CODE_LENGTH, HEIGHT_SIZE, ROOT_SIZE, MAX_SIZE } from './constant.js'
export * from './type.js'
export { code, CODE_LENGTH, HEIGHT_SIZE, ROOT_SIZE, MAX_SIZE }

const bytecode = base64.base64.baseDecode(byteEncodedString)
await load(bytecode.buffer)
```

this fails:

```js
import byteEncodedString from '../gen/wasm_bg.js'

import load, { create } from '../gen/wasm.js'
import { code, CODE_LENGTH, HEIGHT_SIZE, ROOT_SIZE, MAX_SIZE } from './constant.js'
export * from './type.js'
export { code, CODE_LENGTH, HEIGHT_SIZE, ROOT_SIZE, MAX_SIZE }

const buffer = await new Blob([atob(byteEncodedString)]).arrayBuffer() 
await load(buffer)

export { create }
```

Both fail with same error:

```
ncaught CompileError: WebAssembly.instantiate(): Wasm code generation disallowed by embedder
    at null.<anonymous>
  (file:///Users/vcs/git/web3-storage/piece-compute-worker/node_modules/fr32-sha2-256-trunc254-padded-binary-tree-multihash/gen/wasm.js:194:44)
  in __wbg_load
    at null.<anonymous>
  (file:///Users/vcs/git/web3-storage/piece-compute-worker/node_modules/fr32-sha2-256-trunc254-padded-binary-tree-multihash/gen/wasm.js:276:40)
  in __wbg_init
   [code: 10021]
```

so, on line `const instance = await WebAssembly.instantiate(module, imports)`